paasta_tools/utils.py

```diff
@@ -1548,3 +1548,7 @@ def __exit__(cls, *args, **kwargs):
             cls.zk.stop()
             cls.zk.close()
             cls.zk = None
+
+
+class DontKillEverything(Exception):
```

Comment:
you can put this back in ``cleanup_marathon_jobs``, it's not really generalizable. Further, can you name it properly ``DontKillEverythingError`` please?